packages/hardhat-core/test/internal/hardhat-network/stack-traces/execution.ts

```diff
@@ -13,7 +13,7 @@ const senderPrivateKey = Buffer.from(
 const senderAddress = privateToAddress(senderPrivateKey);
 
 export async function instantiateVm(): Promise<VM> {
-  const account = Account.fromAccountData({ balance: 1e18 });
+  const account = Account.fromAccountData({ balance: 1e15 });
```

Comment:
Was this actually necessary?

Reply:
We discussed this on a call. Yes, it is necessary, as exhibited by the CI failure for [this commit](https://github.com/nomiclabs/hardhat/pull/2124/commits/b037daff1ff2815758f526974591db813ad89558), and as explained in the commit message for [this other commit](https://github.com/nomiclabs/hardhat/pull/2124/commits/18a770ac82a340bc65c27d27d7e7adea977bb217).